configs/katello_36.yaml

```diff
@@ -0,0 +1,181 @@
+---
+:project: katello
+:github_org: katello
+:prior_releases:
+  :3.5.0:
+    :redmine_release_id: 250
+  :3.5.1:
+    :redmine_release_id: 329
+:releases:
+  :3.6.0:
+    :redmine_release_id: 284
+:code_name: Imperial IPA
+:repos:
+  :katello:
+    :main: true
+    :branch: KATELLO-3.6
+    :repo: https://github.com/Katello/katello.git
+    :version_branch: true
+  :katello-selinux:
+    :branch: 3.0.3
+    :repo: https://github.com/Katello/katello-selinux.git
+  :katello-installer:
+    :branch: KATELLO-3.6
+    :repo: https://github.com/Katello/katello-installer.git
+    :version_branch: true
+  :hammer-cli-katello:
+    :branch: 0.11.4
+    :repo: https://github.com/Katello/hammer-cli-katello.git
+  :foreman-packaging:
+    :branch: rpm/1.17
+    :repo: https://github.com/theforeman/foreman-packaging.git
+    :version_branch: true
+  :katello-agent:
+    :branch: 3.1.0
+    :repo: https://github.com/Katello/katello-agent.git
+  :bastion:
+    :branch: 6.1.0
+    :repo: https://github.com/Katello/bastion
+  :puppet-pulp:
+    :branch: 5.1.0
+    :repo: https://github.com/Katello/puppet-pulp
+  :puppet-foreman_proxy_content:
+    :branch: 6.0.0
+    :repo: https://github.com/Katello/puppet-foreman_proxy_content
+  :puppet-katello:
+    :branch: 7.0.0
+    :repo: https://github.com/Katello/puppet-katello
+  :puppet-candlepin:
+    :branch: 4.0.0
+    :repo: https://github.com/Katello/puppet-candlepin
+  :puppet-certs:
+    :branch: 3.0.1
+    :repo: https://github.com/Katello/puppet-certs
+
+:gpg_key: ''
+:tags:
```

Comment:
I know I said to look at nightly, but a few things this still needs are at:

https://github.com/theforeman/tool_belt/blob/master/configs/katello_35.yaml#L58

 * Add the mash scripts section
 * Double check the tag names in here, e.g. `katello-thirdparty-candlepin-rhel7` should be `katello-3.6-thirdparty-candlepin-rhel7`